tests/profiling/test_main.py

```diff
@@ -15,11 +15,14 @@ def test_call_script(monkeypatch):
     # Set a very short timeout to exit fast
     monkeypatch.setenv("DD_PROFILING_API_TIMEOUT", "0.1")
     monkeypatch.setenv("DD_PROFILING_ENABLED", "1")
-    stdout, stderr, exitcode, pid = call_program(
-        "ddtrace-run", os.path.join(os.path.dirname(__file__), "simple_program.py")
+    stdout, stderr, exitcode, _ = call_program(
+        "ddtrace-run", sys.executable, os.path.join(os.path.dirname(__file__), "simple_program.py")
     )
-    assert exitcode == 42
-    hello, interval, stacks = stdout.decode().strip().split("\n")
+    if sys.platform == "win32":
+        assert exitcode == 0, (stdout, stderr)
```

Comment:
Excellent question. I digged a lot on this, and I'm not 100% sure of what's wrong. It seems "exit code" and "error level" are different concepts in Windows to my understanding and what is returned by subprocess might be an exit code and not an error level (or vice versa, I don't remember exactly).

Reply:
Worth digging into in the future, since it isn't clear to me at least where this is getting messed up. `ddtrace-run` should be calling `sys.exit(0)` on success, so maybe an issue with our `call_program`?